src/EgressClient.ts

```diff
@@ -74,6 +74,15 @@ export interface TrackCompositeOptions {
   encodingOptions?: EncodingOptionsPreset | EncodingOptions;
 }
 
+/**
+ * Used to supply multiple outputs with an egress request
+ */
+export interface EncodedOutputs {
+  fileOutput?: EncodedFileOutput | undefined;
```

Comment:
nit: would be more economical to drop the output. i.e.

```suggestion
  file?: EncodedFileOutput | undefined;
```

In the future, we could make it `file?: EncodedFileOutput | EncodedFileOutput[]` if we wanted to handle multiple destinations.